readme.txt

```diff
@@ -75,6 +75,7 @@ There are opportunities for developers at all levels to contribute. [Learn more
 
 == Changelog ==
 
+

```

Comment:
Branch currently has conflicts due to this line break so it should be removed since it adds nothing relevant to the code.